docs/kubernetes.md

```diff
@@ -115,10 +115,29 @@ After completing this walkthrough you will know how to:
   3. browse around and explore your pods and services.
   ![Image of Kubernetes dashboard](images/kubernetes-dashboard.png)
 
-# Learning More
+## Troubleshooting
+
+If your cluster is not reachable, you can run the following command to check for common failures.
+
+### Misconfigured Service Principal
+
+If your Service Principal is misconfigured, none of the Kubernetes components will come up in a healthy manner.
+You can check to see if this the problem:
+
+```shell
+ssh -i ~/.ssh/id_rsa cole@colemick-testcluster0.westus2.cloudapp.azure.com sudo journalctl -u kubelet | grep --text autorest

```

Comment:
Do we want to put something more generic than this? Like <user>@<master FQDN>, instead of  cole@colemick-testcluster0.westus2.cloudapp.azure.com 
Possibly something like we do in this comment
MASTERFQDN is obtained in step1
scp azureuser@MASTERFQDN:.kube/config .